tests/test_nft.py

```diff
@@ -0,0 +1,84 @@
+"""
+Copyright (c) Hathor Labs and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+import unittest
+
+from hathorlib import TokenCreationTransaction, Transaction, TxOutput
+from hathorlib.scripts import DataScript
+from hathorlib.utils import is_nft_creation
+
+
+class HathorNFTTestCase(unittest.TestCase):
+    def test_is_nft(self):
+        # Normal tx
+        data = bytes.fromhex('000100010100c994a3f1b46ddeb7134f65cb18b1b11ca7e19d59875a704b2bb2f79f6700b60000694630440'
+                             '220066d379c43ee73c3704730a44d66a077fb2b1cee2b399cbcf87f34d2b2d84308022032e0a93662094c5d'
+                             'b4ed022708981717d06038924535257d181c2fa9f62a6ff9210310a7cd9cae728ddf8c7fef342f963b1cab1'
+                             '97d97b28124ebbd0208d60d9f08780000000200001976a914e7c8133e7611a0ef57830f4321661ff9e5c42f'
+                             '4188ac40200000218def41612cefe10200002d0403a9e39e8176b2e8ca6728f7c8393cea3403f4432c047e5'
+                             'b28cb0470009ed2ab70b799729bcdbaa8edc064bd78fb258ea23fe6688272acad587445ab0000000c')
+        tx = Transaction.create_from_struct(data)
```

Comment:
I think we should always use `tx_or_block_from_bytes` when parsing raw data. Because `BaseTransaction.create_from_struct` implementations don't check the version.

This comment applies for all the cases below.

Reply:
It makes sense, I had to add this method to the lib because we didn't have it. https://github.com/HathorNetwork/python-hathorlib/pull/6/commits/f40526509137c9d2ce821028d4f4dc3992b7e31f

Reply:
Oh, right, this is so similar to `hathor-core` that I forgot. Which is great, migrating shouldn't be too painful.